dist/index.html

```diff
@@ -13,11 +13,14 @@
     <header>
       <nav class =top-nav>
         <h1 class='travel-title'>travel.</h1>
-        <form class='login '>
-          <input id='userName' aria-label='username' class='user-name' type='text' name='username' placeholder='username' value='' required>
-          <input id='password' aria-label='password' class='password' type='password' name='password' placeholder='password' value='' required>
-          <button id='login-button' aria-label='login-button' type='button' class='login-button' name='button'>enter.</button>
+        <form class='login' id='loginForm'>
+          <input id='userName' aria-label='username' class='user-name' type='text' name='username' placeholder='username' required>
```

Comment:
Not sure what function the name attribute serves? I do not have it. Maybe it's instead of a label? Worth investigating more *Kyra says to herself*

Reply:
Lol, You're right, I'm actually not sure what purpose the name attribute has either. I'll look into that. Maybe an additional screen reader thing?